src/status_im/ui/components/status_bar/styles.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im.ui.components.status-bar.styles
+  (:require [status-im.ui.components.styles :as styles]
+            [status-im.utils.platform :as platform])
+  (:require-macros [status-im.utils.styles :refer [defstyle]]))
+
+;; :main
+(defstyle status-bar-main
+  {:ios     {:background-color styles/color-white
+             :bar-style        "default"}
+   :android {:background-color "transparent"
+             :translucent      true
+             :bar-style        "dark-content"}})
+
+(def view-main
+  {:background-color styles/color-white
+   :height           (get platform/platform-specific :status-bar-default-height)})
+
+;; :transparent
+(defstyle status-bar-transparent
+  {:bar-style "light-content"
+   :ios       {:background-color styles/color-transparent}
+   :android   {:background-color "transparent"
+               :translucent      true}})
+
+(def view-transparent
+  {:background-color styles/color-transparent
+   :height           (get platform/platform-specific :status-bar-default-height)})
+
+;; :modal
+(defstyle status-bar-modal
+  {:bar-style "light-content"
+   :ios       {:background-color "#2f3031"}
+   :android   {:background-color styles/color-black}})
+
+(defstyle view-modal
+  {:ios     {:background-color "#2f3031"
+             :height           (get platform/platform-specific :status-bar-default-height)}
+   :android {:background-color styles/color-black
+             :height           0}})
+
+;; :modal-white
+(defstyle status-bar-modal-white
+  {:ios     {:background-color styles/color-white
+             :bar-style        "default"}
+   :android {:background-color styles/color-black
+             :bar-style        "light-content"}})
+
+(defstyle view-modal-white
+  {:ios     {:background-color styles/color-white
+             :height           (get platform/platform-specific :status-bar-default-height)}
+   :android {:background-color styles/color-black
+             :height           0}})
+
+;; :modal-wallet
+(defstyle status-bar-modal-wallet
+  {:bar-style "light-content"
+   :ios       {:background-color styles/color-blue4}
+   :android   {:background-color styles/color-black}})
+
+(defstyle view-model-wallet
+  {:ios     {:background-color styles/color-blue4
+             :height           (get platform/platform-specific :status-bar-default-height)}
```

Comment:
There are multiple calls to `(get platform/platform-specific :status-bar-default-height)` in this ns. Can you introduce a loval var?